composer.json

```diff
@@ -12,13 +12,18 @@
     "require": {
         "php": "^7.3",
         "nexylan/slack": "^3.0",
+        "php-http/discovery": "^1.6",
         "php-http/httplug-bundle": "^1.17",
+        "php-http/httplug": "^2.0",
         "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0"
     },
     "require-dev": {
+        "http-interop/http-factory-guzzle": "^1.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
+        "php-http/guzzle6-adapter": "^1.1.1 || ^2.0",
```

Comment:
these 2 guzzle-related things were because we needed SOME valid httplug implementation so that the normal "discovery" could work inside the integration test.